bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,337 @@
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Ampere</label>
+		<state pattern="%.2f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="volts">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Ampere</label>
+		<state pattern="%.0f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="image">
+		<item-type>Image</item-type>
+		<label>Image</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="schedule-start-time">
+		<item-type>DateTime</item-type>
+		<label>Schedule Time</label>
+		<state pattern="%1$tH:%1$tM"/>
+		<command>
+			<options>
+				<option value="16:00">16:00</option>
+				<option value="16:30">16:30</option>
+				<option value="17:00">17:00</option>
+				<option value="17:30">17:30</option>
+				<option value="18:00">18:00</option>
+				<option value="18:30">18:30</option>
+				<option value="19:00">19:00</option>
+				<option value="19:30">19:30</option>
+				<option value="20:00">20:00</option>
+			</options>
+		</command>
+	</channel-type>
+	<channel-type id="schedule-end-time">
+		<item-type>DateTime</item-type>
+		<label>Schedule Time</label>
+		<state pattern="%1$tH:%1$tM"/>
+		<command>
+			<options>
+				<option value="04:00">4:00</option>
+				<option value="04:30">4:30</option>
+				<option value="05:00">5:00</option>
+				<option value="05:30">5:30</option>
+				<option value="06:00">6:00</option>
+				<option value="06:30">6:30</option>
+				<option value="07:00">7:00</option>
+				<option value="07:30">7:30</option>
+				<option value="08:00">8:00</option>
+			</options>
+		</command>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="datetime">
+		<item-type>DateTime</item-type>
+		<label>Date Time</label>
+		<state pattern="%1$tA, %1$td.%1$tm. %1$tH:%1$tM"/>
+	</channel-type>
+	<channel-type id="scene-trigger">
+		<item-type>Number</item-type>
+		<label>Scene Trigger</label>
+		<command>
+			<options>
+				<option value="0">Trigger</option>
+				<option value="1">Undo</option>
+			</options>
+		</command>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="pm25">
+		<item-type>Number:Density</item-type>
+		<label>Particulate Matter category 2.5</label>
+		<state pattern="%.0f µg/m³" readOnly="true"/>
+	</channel-type>
+	<channel-type id="voc">
+		<item-type>Number:Density</item-type>
+		<label>VOC Index</label>
+		<state pattern="%.0f mg/m³" readOnly="true"/>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<state pattern="%.1f %%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="time">
+		<item-type>Number:Time</item-type>
+		<label>Time</label>
+	</channel-type>
+	<channel-type id="duration">
+		<item-type>Number:Time</item-type>
```

Comment:
The unit for Number:Time defaults to seconds. I wonder if the option values work as how they are currently defined, just not sure, please double check if not already done.

Reply:
Checked. 
It works with the `value` definition with `number` and `quantity`. So code receives a QuantityType which I convert into seconds. 
`<option value="1 min">1 minute</option>`
If a raw number is sent it will be taken as seconds. I described this now better in readme.